portfolio/src/main/webapp/script.js

```diff
@@ -93,14 +94,14 @@ function scrollToElement(elementId, duration = 300)
 /**
  * Function that loads comments
  */
-function LoadComments() {
+function loadComments() {
     // Create static commentsLoaded variable, if not exists
-    if(typeof LoadComments.commentsLoaded == 'undefined') {
-        LoadComments.commentsLoaded = 0;
+    if(typeof loadComments.commentsLoaded == 'undefined') {
+        loadComments.commentsLoaded = 0;
     }
```

Comment:
Maybe you could use '===' instead of '==' here. Please see some examples [here](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons)

Reply:
Right, thanks! Resolved